web/app.py

```diff
@@ -0,0 +1,47 @@
+import os
+import tempfile
+import streamlit as st
+import mikeio
+import fmskill
+
+"""
+# FMskill Drag and Drop
+"""
+
+tmp_folder = tempfile.mkdtemp()
+
+with st.sidebar:
+    
+    obs = st.file_uploader("Observation", type="dfs0")
+
+    if obs:
+        fn_obs = os.path.join(tmp_folder, "obs.dfs0")
+        with open(fn_obs, "wb") as f:
+            f.write(obs.getvalue())
+
+        dfs = mikeio.open(fn_obs)
+        items = [item.name for item in dfs.items]
+        obs_item = st.selectbox(label="Item", options=items)
+
+    mod = st.file_uploader("Model", type="dfs0")
+
+    if mod:
+        fn_mod = os.path.join(tmp_folder, "mod.dfs0")
+        with open(fn_mod, "wb") as f:
+            f.write(mod.getvalue())
+        dfs = mikeio.open(fn_mod)
+        items = [item.name for item in dfs.items]
+        mod_item = st.selectbox(label="Item", options=items)
+
+if mod and obs:
+
+    c = fmskill.compare(fn_obs, fn_mod, obs_item=obs_item, mod_item=mod_item)
```

Comment:
@ecomodeller , I see that the entry point to `fmskill` is here. So is it correct that spatial information (location of station) is not required / not accommodated for now? I.e., the focus is here to compare to time series, and it's the user's responsibility that it makes sense to compare those two time series?

Reply:
P.S.: I do like that flexibility.

Reply:
Don't take it too seriously just yet. It was made after a discussion with @jsmariegaard on the quickest way to do ad-hoc comparisons.